src/node_version.h

```diff
@@ -112,10 +112,11 @@
  * V8 6.9: 66
  * V8 7.0: 67
  * V8 7.1: 68
+ * V8 7.3: 70
  *
  * More information can be found at https://nodejs.org/en/download/releases/
  */
-#define NODE_MODULE_VERSION 68
+#define NODE_MODULE_VERSION 70
```

Comment:
Maybe bump to 71 in light of https://github.com/nodejs/TSC/issues/651#issuecomment-471807498?

Reply:
Bumped to 71 (cc @rvagg)

Reply:
ooof, sorry, I should have checked for a PR like this when I wrote that. Thanks for accommodating the change @targos